plugin.py

```diff
@@ -31,6 +37,15 @@ def arch() -> str:
         raise RuntimeError("Unknown architecture: " + sublime.arch())
 
 
+def get_setting(view: sublime.View, key: str, default: Optional[str] = None) -> Union[bool, str]:
```

Comment:
- The third argument should be `Any` type, I believe. You are passing `False` from all places.
- The return type doesn't match what `settings.get` can return.

I'd suggest installing LSP-pyright to get those errors visualized.

Reply:
I did try! But Where would LSP-pyright resolve sublime and the LSP package from.

Reply:
It has `"pyright.dev_environment": "sublime_text"` option that does that.